x-pack/plugins/security_solution/common/types/app_features.ts

```diff
@@ -10,6 +10,14 @@ export enum AppFeatureSecurityKey {
    * Enables Advanced Insights (Entity Risk, GenAI)
    */
   advancedInsights = 'advanced_insights',
+  /**
+   * Enables Endpoint Response Actions like isolate host, trusted apps, blocklist, etc.
+   */
+  endpointResponseActions = 'endpoint_response_actions',
```

Comment:
Are these values final? or just preliminary until we get around to creating them in accordance to the PLI spreadsheet?

Reply:
These appFeatures represent the PLI in the spreadsheet, they are final. However, I am not familiar enough with the endpoint logic to be 100% sure I implemented the correct scope, if you guys think we should include something else, or something less, or change the names of the flags, I'll be happy to adapt it.

Reply:
Ok. We'll likely adjust it. I'm thinking we create an individual item for each response action rather than combine them all into one. that gives us the freedom to ensure that future Response actions can be used in different PLI's if any new ones come up.

Reply:
There will be only one `AppFeatureKey` for each PLI in the spreadsheet, the granularity should be configured at the Kibana feature level here: 
https://github.com/elastic/kibana/blob/4d9f0c3a6f0c926a733b0fc113dfc20fe1223e07/x-pack/plugins/security_solution/server/lib/app_features/security_kibana_features.ts#L170-L198